README.md

```diff
@@ -34,23 +40,40 @@
 
 # Why go left
 
-LeftWM is a tiling window manager written in [Rust](https://github.com/rust-lang/rust) that aims to be stable and performant. LeftWM is [designed to do one thing and to do that one thing well](https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well): _be a window manager_. LeftWM therefore follows the following mantra:
+LeftWM is a tiling window manager written in [Rust] that aims to be stable and performant. LeftWM is
+[designed to do one thing and to do that one thing well][unix-philosophy]: _be a window manager_.
+LeftWM therefore follows the following mantra:
 
 > LeftWM is not a compositor.  
 > LeftWM is not a lock screen.  
-> LeftWM is not a bar. But, there are lots of good bars out there. With themes, picking one is as simple as setting a symlink.
+> LeftWM is not a bar. But, there are lots of good bars out there. With themes, picking one is as
+  simple as setting a symlink.
 
-Because you probably want more than just a black screen, LeftWM is built around the concept of themes. With themes, you can choose between different bars, compositors, backgrounds, colors, docks, and whatever else makes you happy.
+Because you probably want more than just a black screen, LeftWM is built around the concept of
+themes. With themes, you can choose between different bars, compositors, backgrounds, colors, docks,
+and whatever else makes you happy.
 
-LeftWM was built from the very beginning to support multiple screens and ultrawide monitors. The default keybindings support ultrawide monitors and multiple screens.
+LeftWM was built from the very beginning to support multiple screens and ultrawide monitors. The
+default keybindings support ultrawide monitors and multiple screens.
```

Comment:
The scheme I used to break lines is not based on sentences, it's as many words can fit on a line such that they don't exceed 100 characters in length. I don't like to start breaking on sentence structure because then you end up worrying a bunch about where you should put line breaks. You inevitably get into situations where if you break one word early in sentence 1 then sentence 2 becomes too long and now you have to decide where to break it - it's just messy.

Reply:
As above, it seems like github displaying the diff lead me to the false conclusion that things were getting messy.
Looking at the raw file it looks just fine..